test/fixtures/help-texts/loopback_app_help.txt

```diff
@@ -35,4 +36,4 @@ Example:
 
 Available generators:
 
-  loopback:app
+
```

Comment:
This change looks skeptical

Reply:
@shimks 
> loopback:app

That looks like a workaround to hack the old code. The available generators should be detected by [code](https://github.com/strongloop/generator-loopback/blob/78b61bcb0ebdde746350d7a02161a5fcfe070c0f/app/index.js#L75-L83), I don't remember why there was an extra `loopback:app` there. 
I think removing it is the expected behaviour.